internal/devbox/generate/tmpl/DevboxImageDockerfile

```diff
@@ -1,7 +1,7 @@
 FROM debian:stable-slim
 
 # Optional arg to install custom devbox version
-ARG DEVBOX_USE_VERSION
+ARG DEVBOX_VERSION
```

Comment:
This has to be different from `ENV DEVBOX_USE_VERSION` otherwise docker ignores the ENV